modules/project_services/outputs.tf

```diff
@@ -16,9 +16,6 @@
 
 output "project_id" {
   description = "The GCP project you want to enable APIs on"
-  value = element(
-    concat(google_project_service.project_services.*.project, [""]),
-    0,
-  )
+  value       = var.enable_apis && length(var.activate_apis) > 0 ? var.project_id : ""
```

Comment:
We still need this to pass through the project_id from the `google_project_service` resource (to preserve the dependency graph), not directly from var.project_id.